firmware/config/stm32f7ems/chconf.h

```diff
@@ -41,12 +41,13 @@
  *
  */
 
-#define PORT_IDLE_THREAD_STACK_SIZE     1024
+#define PORT_IDLE_THREAD_STACK_SIZE     32
 
-// rusEfi main processing happens on IRQ so PORT_INT_REQUIRED_STACK has to be pretty large.
-// see also a strange comment about PORT_INT_REQUIRED_STACK in global_shared.h
+// PORT_INT_REQUIRED_STACK is the stack space added to every thread's stack so that even if the thread
```

Comment:
Actually we clearly have a copy-paste of a large comment and you are now editing the copy-paste in three places? is not this a bit weird? I wonder if global_shared.h is the better SINGLE place for this comment, with other files potentially referring people to that single centralized comment?

Reply:
yes - probably true, I'll do that